src/app/src/main/kotlin/org/imperial/mrc/hint/models/ModelRunParameters.kt

```diff
@@ -0,0 +1,6 @@
+package org.imperial.mrc.hint.models
+
+data class ModelRunParameters(val maxIterations: Int,
+                              val noOfSimulations: Int,
+                              val sleep: Int,
```

Comment:
So this sleep is an artefact of us not having Jeff's model yet. The model atm just sleeps for specified time and returns 2. If that adds debt I can update the model so that this isn't a required param.

Reply:
I mean, this is going to have to change either way so I wouldn't worry for now. As and when we get the real interface I will update this accordingly.